heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java

```diff
@@ -133,6 +133,7 @@ public HeronExecutorTask(final REEFFileNames fileNames,
         env,
         topologyJar,
         topologyDefFile,
+        false,
```

Comment:
because in this class, no verbose value has been set. I just put `false` here. cc @ashvina do you have any suggestion?